agents-docker-compose.yml

```diff
@@ -0,0 +1,74 @@
+services:
+  message_queue:
+    build: ./src/message_queue
+    ports:
+      - "8100:8100"
+    environment:
+      - MESSAGE_QUEUE_HOST=message_queue
+      - MESSAGE_QUEUE_PORT=8100
+    networks:
+      - ragapp-network
+
+  control_plane:
+    build: ./src/control_plane
+    ports:
+      - "8000:8000"
+    environment:
+      - CONTROL_PLANE_HOST=control_plane
+      - CONTROL_PLANE_PORT=8000
+      - MESSAGE_QUEUE_URL=http://message_queue:8100
+      - OPENAI_API_KEY=${OPENAI_API_KEY}
+      - CONTROL_PLANE_MODEL=gpt-3.5-turbo
+      - CONTROL_PLANE_ORCHESTRATOR_TYPE=pipeline
```

Comment:
1. pipeline also needs OPENAI_KEY?

2. if we use pipeline, then we should configure it here. e.g.:
```
PIPELINE=[travel_agent,research_agent]
```

Reply:
> pipeline also needs OPENAI_KEY?

I also have this question too, but it requires construct a vector store engine there which require embedding model, so i still let the openapi there to easier testing.

> if we use pipeline, then we should configure it here. e.g.:

Yes, i thought about this but we cannot do that now. The reason is we need construct the orchestrator to launch the control_plane where those agents need the control_plane booting first. That's why i just hard-code the pipeline chain for now. 
Another thing is it seems that we just need a service definition to init the pipeline orchestrator so if we have a share config for the service then we would able to do that.

Reply:
you can share the config using `configs`

but if this pipelineorchestrator is to difficult to set up, how about just using the agentorchestrator?